doc/classes/OS.xml

```diff
@@ -201,7 +201,12 @@
 			<return type="String">
 			</return>
 			<description>
-				Returns the host OS locale.
+				Returns the host OS locale as a string of the form [code]language_Script_COUNTRY_VARIANT@extra[/code].
+				[code]language[/code] - 2 or 3 letter [url=https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes]language code[/url], in lower case.
+				[code]Script[/code] - optional, 4 letter [url=https://en.wikipedia.org/wiki/ISO_15924]script code[/url], in title case.
+				[code]COUNTRY[/code] - optional, 2 or 3 letter [url=https://en.wikipedia.org/wiki/ISO_3166-1]country code[/url], in upper case.
+				[code]VARIANT[/code] - optional, language variant, region and sort order. Variant can have any number of underscored key words.
+				[code]extra[/code] - optional, semicolon separated list of additional key words. Currency, calendar, sort order and numbering system information.
```

Comment:
I'm guessing some of those are OS-specific, maybe this should be documented too? Or do all OSes follow the same format aside from some using `-` and others using `_`?

Reply:
The overall pattern is valid for all OSes, but none of the locales have all parts. Only `@` is Linux specific, the rest can be encountered on any OS.